package.json

```diff
@@ -17,9 +17,9 @@
     "url": "https://github.com/su-sws/decanter.git"
   },
   "version": "0.1.0",
-  "dependencies": {
-    "bourbon": "~5.0.0-beta.7",
-    "bourbon-neat": "^2.0.0-beta.2",
-    "font-awesome": "^4.6.3"
+  "devDependencies": {
+    "bourbon": "5.0.0-beta.7",
+    "bourbon-neat": "2.0.0-beta.2",
```

Comment:
That makes sense to me and I think it's a safe path moving forward. Thoughtbot tends to be (sometimes a little too) conservative with their releases, and so far I haven't run into any breaking changes that haven't been announced.